README.md

```diff
@@ -1,26 +1,39 @@
 # Grafana Status Panel
 
 This is a plugin meant to be used as a centralized view for the status of a server/cluster in a glance.
-It is very similar to the Single Stat panel, but it can hold multiple values and you can customize the thresholds for each of them separately.
+It is very similar to the Single Stat panel, but it can hold multiple values pulled from different data sources.
+Each value can be used to customize the panel in different ways: 
+* Mark the severity of the server
+* Mark if the server is disabled
+* Show extra data in the panel about the server  
 
 ## The General Idea
 Lets say that you want to monitor a bunch of servers, and you want to keep track of multiple stats for each of them, and see in a glance the status of all of them.
 
-This plugin will make it easier to do. You just add all the measurements that you want to track, set the threshold for each (you could also repeat the panel on a template if you have multiple instances that you want to watch) and you get an overview that will report to you if there is anything wrong with any measurements. That means that if all the measurements are is the OK zone, the panel will be green. if evan one of the measurements is in the WARNING zone, it will be yellow, and red if any of them is CRITICAL.
+This plugin will make it easier to do. You just add all the measurements you want to track, and choose how you want their values to be treated:
+1. Server severity marker - Set the threshold for each and you get an overview that will report to you if there is anything wrong with any measurements. That means that if all the measurements are in the OK zone, the panel will be green. If evan one of the measurements is in the WARNING zone, it will be yellow, and red if any of them is CRITICAL.
+2. Server disable marker - Set the exact value that represent if the server is disabled, the panel will be grey.
+3. Annotation - show extra information about the server in the top left side of the panel.
+
```

Comment:
This plugin is not just for server monitoring, but monitoring anything that could have different alert states.
Please change the documentation to something more generic.